employee.py

```diff
@@ -30,13 +29,13 @@ def __init__(self, attributes):
                             job_family: 10 --> Intern-Technical
                             ...
         """
-        self.id = attributes.get("id")
-        self.department = attributes.get("department")
-        self.cost_center = attributes.get("cost_center")
-        self.manager_id = attributes.get("manager_id")
-        self.location = attributes.get("location")
-        self.lowest_dir_id = attributes.get("lowest_dir_id")
-        self.job_family = attributes.get("job_family")
+        self.id = int(float(attributes.get("id")))
+        self.department = int(float(attributes.get("department")))
+        self.cost_center = int(float(attributes.get("cost_center")))
+        self.manager_id = int(float(attributes.get("manager_id")))
+        self.location = int(float(attributes.get("location")))
+        self.lowest_dir_id = int(float(attributes.get("lowest_dir_id")))
+        self.job_family = int(float(attributes.get("job_family")))
```

Comment:
This is useful, just have to make sure that we account for nulls when we use it now. I think its good thought because it will create an obvious error so it will be clear what is wrong if we forget to at some point later on. Also, as you mentioned its easy to just eleminate all the nulls with pandas.